src/main/java/de/teamlapen/vampirism/player/vampire/VampirePlayer.java

```diff
@@ -1018,6 +1077,7 @@ protected void writeFullUpdate(NBTTagCompound nbt) {
         nbt.setInteger(KEY_EYE, getEyeType());
         nbt.setInteger(KEY_FANGS, getFangType());
         nbt.setBoolean(KEY_GLOWING_EYES, getGlowingEyes());
+//        nbt.setInteger(KEY_VICTIM_ID, victim);
```

Comment:
Just to be sure I would leave this active

Reply:
yeah I commented it because when I made that change something was broken and I never uncommented it.  its fixed now